controller/internal/enforcer/nfqdatapath/datapath.go

```diff
@@ -725,7 +725,7 @@ func (d *Datapath) EnableDatapathPacketTracing(contextID string, direction packe
 		d.packetTracingCache.Remove(contextID) // nolint
 	}()
 
-	return nil
+	return ctx.Err()
```

Comment:
Why?

Reply:
let me change that ... it's not really helping/doing anything anymore at that return